components/eamxx/src/share/atm_process/atmosphere_process_hash.cpp

```diff
@@ -129,7 +129,7 @@ ::print_global_state_hash (const std::string& label, const bool in, const bool o
   if (m_comm.am_i_root())
     for (int i = 0; i < nslot; ++i)
       if (show[i])
-        fprintf(stderr, "exxhash> %4d-%9.5f %1d %16llx (%s)\n",
+        fprintf(stderr, "exxhash> %4d-%9.5f %1d %16lx (%s)\n",
```

Comment:
There is another `%16llx` below that needs to be reverted to `%16lx`. I think it must show up in your environment's warnings.